src/pages/GoalsSelector/GoalsSelector.js

```diff
@@ -46,34 +59,17 @@ const GoalsSelector = () => {
             cover crops. The first goal you select will have the highest priority in sorting and then
             decrease for each additional goal. Hover on a goal for more information.
           </Typography>
-          {allGoals.length === 0 ? (
-            <div className="goals col-lg-12">
-              <div className="row">
-                <div className="col-3">
-                  <Skeleton style={goalSkeletonStyle} />
-                </div>
-                <div className="col-3">
-                  <Skeleton style={goalSkeletonStyle} />
-                </div>
-                <div className="col-3">
-                  <Skeleton style={goalSkeletonStyle} />
-                </div>
-                <div className="col-3">
-                  <Skeleton style={goalSkeletonStyle} />
-                </div>
-              </div>
-            </div>
-          ) : (
+          {allGoals.length > 0 && (
             <Grid container spacing={4} className="goals" style={{ justifyContent: 'center' }}>
-              {allGoals[0].fields ? (
+              {allGoals.length > 0 ? (
```

Comment:
this is guaranteed to be true by line 62 reight?

Reply:
good catch!